diff --git a/node_modules/react-native-libsodium/react-native-libsodium.podspec b/node_modules/react-native-libsodium/react-native-libsodium.podspec
index b9b9308..72899cc 100644
--- a/node_modules/react-native-libsodium/react-native-libsodium.podspec
+++ b/node_modules/react-native-libsodium/react-native-libsodium.podspec
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
 
   s.source_files = "ios/**/*.{h,m,mm}", "cpp/**/*.{h,cpp}"
 
-  s.vendored_frameworks = "libsodium/build/libsodium-apple/Clibsodium.xcframework"
+  s.dependency "libsodium"
 
   s.dependency "React-Core"
 
